webapp/src/components/tutorial/TutorialContainer.tsx

```diff
@@ -106,9 +109,77 @@ export function TutorialContainer(props: TutorialContainerProps) {
     const markdown = steps[visibleStep].headerContentMd;
     const hintMarkdown = steps[visibleStep].hintContentMd;
 
+    const validateTutorialStep = async () => {
+        const globalValidators = tutorialOptions.globalValidationConfig?.validators
+                                    ?? PopulateValidatorCache(tutorialOptions.globalValidationConfig);
+        const localValidators = currentStepInfo.localValidationConfig?.validators
+                                    ?? PopulateValidatorCache(currentStepInfo.localValidationConfig);
+
+        let validators: pxt.Map<CodeValidator> = {}
+        if(localValidators) {
```

Comment:
```suggestion
        if (localValidators) {
```

nit, several of these throughout file